LockGoogleApp/Info.plist

```diff
@@ -3,9 +3,9 @@
 <plist version="1.0">
 <dict>
 	<key>Auth0ClientId</key>
-	<string>U5MhUrbyQHSVWjlEqZSTCBUFABLbJAS3</string>
+	<string>YN9SCHpqf8uLRpZbkLfehgmqlAObPgXL</string>
 	<key>Auth0Domain</key>
-	<string>overmind.auth0.com</string>
+	<string>brucke.auth0.com</string>
```

Comment:
We should not commit clientid of this tenant (removed it from dashboard already) so put back the old ones